casadi/interfaces/gurobi/gurobi_interface.cpp

```diff
@@ -159,6 +159,8 @@ namespace casadi {
     casadi_assert(!flag && m->env, "Failed to create GUROBI environment. Flag: " + str(flag)
       + ":" + GRBgeterrormsg(m->env));
 
+    m->pool_sol_nr = 0;
```

Comment:
Is this supposed to be an option?

Reply:
Nope, this is one of the outputs.
We could add an option to receive the pool or not?

Reply:
Indeed I misread. This needs to be optional. However, Gurobi may probably already have an option for this already?

Reply:
Yes, you can limit the number of solutions in the solution pool, but the minimum is always 1.